packages/core/utils/src/index.ts

```diff
@@ -173,7 +174,7 @@ export function deconstructUrl(endpoint) {
   return {
     protocol: matches[1],
     host: matches[2],
-    port: matches[3],
+    port: matches[3] ? parseInt(matches[3], 10) : false,
```

Comment:
Parse port to Integer because we have to do Math with it in the proxy for example